src/clients/customer/lib/api/firestore_api.dart

```diff
@@ -99,4 +99,31 @@ class FirestoreApi {
   CollectionReference getAddressCollectionForUser(String userId) {
     return usersCollection.doc(userId).collection(AddressesFirestoreKey);
   }
+
+  Future<List<Merchant>> getMerchantsCollectionForRegion(
+      {required String regionId}) async {
+    log.i('regionId:$regionId');
+    try {
+      final regionCollections =
+          await regionsCollection.doc(regionId).collection('merchants').get();
+      if (regionCollections.docs.isEmpty) {
+        log.v('We have no merchants in this region');

```

Comment:
Lets change this into a warning log. When writing about it mention that we're printing a warning because there should not be a region with no merchants in it. If that'e the case the document shouldn't exist.

Reply:
Got it